civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -16,6 +16,70 @@ $row-margin-bottom: 15px;
   margin-bottom: 20px;
 }
 
+.chr_leave-request-modal__form {
```

Comment:
Instead of applying the new dropdown styles for `chr_leave-request-modal__form` we should make it more generic as this kind of dropdown would get reused.
So you can create a new class called similar to `chr_ui-select--info-colored`(or something similar)
```html
<ui-select
        class="chr_ui-select--info-colored"
        autofocus
        ...
```

in scss (sites/all/themes/civihr_employee_portal_theme/civihr_default_theme/scss/components/_ui-select.scss)
```scss
.chr_ui-select--info-colored {
  //Custom css
  .ui-select-container{}...
  ...
}